src/wizards/audience/views/content-gates/content-rule-control.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Content Gate component.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import ContentRuleControlTaxonomy from './content-rule-control-taxonomy';
+import { FormTokenField } from '../../../../../packages/components/src';
+
+const noop = () => {};
+
+export default function ContentRuleControl( { slug, value, onChange }: GateContentRuleControlProps ) {
+	const rule = window.newspackAudienceContentGates.available_content_rules[ slug ];
```

Comment:
Do we need to protect against `available_content_rules` or `available_content_rules` being undefined?

```
window.newspackAudienceContentGates?.available_content_rules?.[ slug ];
```